proteus/tests/cylinder2D/ibm_method/mesh.edge

```diff
@@ -1,3 +1,335 @@
-version https://git-lfs.github.com/spec/v1
-oid sha256:3f9c96cdbab38fd8e3a2ee647c41a63731f836028382b1b692460530757c1680
-size 5840
+333  1
```

Comment:
Why put this one under git? Just curious.

Reply:
Actually, this file was inadvertently added. It's not supposed to be here.

Reply:
It was the same error you ran into in which git auto added the file as one that needed to be committed.  

Force pushed same commit but omitting mesh.edge